src/components/PopularItemCarousel/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
+import CarouselItem from './Item'
+
+const PopularItemCarousel = () => {
+  return (
+    <CarouselProvider
+      visibleSlides={3}
+      naturalSlideWidth={400}
+      naturalSlideHeight={500}
+      totalSlides={5}
+      infinite={true}
```

Comment:
To make it reusable, Do you think these `value` should be pass as `props`?

```suggestion
      visibleSlides={value}
      naturalSlideWidth={value}
      naturalSlideHeight={value}
      totalSlides={value}
      infinite={value}
```

Reply:
For now, we'll keep it this way since we don't know how `DRY` could it be but will keep this in mind.